epistemix_platform/src/epistemix_platform/repositories/s3_upload_location_repository.py

```
@@ -29,6 +29,49 @@
 logger = logging.getLogger(__name__)
 
 
+def create_s3_client(region_name: str | None = None, s3_client: Any | None = None) -> Any:
```

Ruff (0.14.1)

Unused function argument: `s3_client`

(ARG001)